### PR TITLE
[stable/elasticsearch-curator] add jobs in hooks, delete .Values.config

### PR DESCRIPTION
**Pull request by kimxogus**
#### What this PR does / why we need it:
1. Add helm hooks to run jobs on install or upgrade.

2. Delete `.Values.config` because I think it only makes confusion between `.Values.config` and `.Values.configMaps.config_yml`.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by gianrubio — COMMENTED** [review]
/ok-to-test
/lgtm

**Comment by Towmeykaw** [comment]
Just for future reference. This was a breaking change and should have bumped the major version.

File: stable/elasticsearch-curator/values.yaml [hunk 1]
```diff
@@ -49,26 +47,26 @@ configMaps:
           epoch:
           exclude: False
   # Having config_yaml WILL override the other config
```
Comment (desaintmartin) [inline stable/elasticsearch-curator/values.yaml:49]: Note: should this be deleted?

Comment (kimxogus) [reply stable/elasticsearch-curator/values.yaml:49]: yes. it seems need to be deleted.

**Pull request merged**